cmd/goexports/goexports.go

```diff
@@ -61,16 +66,19 @@ func init() {
 				"{{$key}}": reflect.ValueOf({{$value.Name}}),
 			{{end -}}
 		{{end}}
-
+		{{- end}}
+		{{- if .Typ}}
```

Comment:
If you have a blank like below this and the `.Wrap` conditional below, you'll get nice stanza formatting for these groups of symbols.

Reply:
actually did the reverse (put the blank line at the end of the block) to better handle the case where, _e.g._ there would be no `.Val` but `.Typ`. (following your recommandation, a blank line between `map[string]reflect.Value{` and `// type definitions` would have been inserted.)

Reply:
This is still not right; if you have `.Val` or `.Typ` but no `.Wrap` you end up with a trailing blank.

The logic, when not done in a loop is actually sort of convoluted. Maybe not caring is better.

Reply:
the file is `gofmt`'d so the trailing blank line should be removed (I think)